### ISSUE TITLE
popupResponse  radio or checkbox causes horizontal scrollbar

### ISSUE THREAD

**Issue by giuliomoro** [BrechtDeMan/WebAudioEvaluationTool#63]
The computation of the `left` parameter in order to center the checkboxes/radio buttons, combined with the fact that the width of the popupResponse div is the same as the container div makes the horizontal scrollbar appear every time you use a radio or checkbox.
There is nothing bad, in my opinion, in having boxes and radio buttons aligned left, so I suggest the following:
````diff
--- a/js/core.js
+++ b/js/core.js
@@ -732,8 +732,8 @@ function interfacePopup() {
                 index++;
                        }
             this.popupResponse.style.textAlign="";
-            var leftP = 50-(((max_w/$('#popupContent').width())/2)*100);
-            this.popupResponse.style.left=leftP+"%";
                } else if (node.specification.type == 'radio') {
             if (node.response == undefined) {
                 node.response = {name: "", text: ""};
@@ -761,8 +761,8 @@ function interfacePopup() {
                     max_w = w;
                        }
             this.popupResponse.style.textAlign="";
-            var leftP = 50-(((max_w/$('#popupContent').width())/2)*100);
-            this.popupResponse.style.left=leftP+"%";
                } else if (node.specification.type == 'number') {
                        var input = document.createElement('input');
                        input.type = 'textarea';
````

**Comment by giuliomoro** [comment]
This is a design changed, so I fixed it only in my branch, b22f843

**Comment by nickjillings** [comment]
Now use tables to perform justification of text and alignment of input nodes.
